services/frontend/pkg/config/config.go

```diff
@@ -117,6 +117,7 @@ type OCS struct {
 	ResourceInfoCaches      ResourceInfoCaches `yaml:"resource_info_caches,omitempty"` // only used for redis
 	CacheWarmupDriver       string             `yaml:"cache_warmup_driver,omitempty"`  // not supported by the oCIS product, therefore not part of docs
 	CacheWarmupDrivers      CacheWarmupDrivers `yaml:"cache_warmup_drivers,omitempty"` // not supported by the oCIS product, therefore not part of docs
+	EnableDenials           bool               `yaml:"enable_denials" env:"FRONTEND_OCS_ENABLE_DENIALS" desc:"EXPERIMENTAL: enable the option to deny access on resources."`
```

Comment:
Reading that this is experimental - ok...
But:  `enable the option to deny access on resources` is very cryptic.
- `Enable the option` means for me, that there is another value that can be manipulated that does the details.
- `Ressources`, hmmm, any hint to be added what is meant, or an example?

Reply:
> Ressources, hmmm, any hint to be added what is meant, or an example?

Resource is used all over the code base and means "file or folder"

> Enable the option

We can say `enable the feature`